### PR TITLE
desktopVersionFix

### PR DESCRIPTION
**Pull request by IgorSinyak**
JS and SCSS code corrctions

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -1,63 +1,85 @@
...
+var navMenu;
+var navList;
+var navMenuHeight;
 
 $(document).ready( function() {
+
   navMenu = $(NAV_BLOCK_ID);
+  navMenuHeight = navMenu.outerHeight();
   navPosition = navMenu.offset().top;
   navList = $(NAV_LIST_ITEM_SELECTOR)
```
Comment (vitali-usik) [inline js/script.js]: Missed `;`

File: js/script.js [hunk 2]
```diff
@@ -1,63 +1,85 @@
...
   
   $(CONTACTS_SELECTOR).click(function() {
     var cont = $(CONTACTS_SELECTOR);
-    var popup = $(POPUP_SELECTOR);  
-    cont.removeClass('active');
-    $(this).addClass('active');
-    var message = $(this).attr('data-popuptext');
-    popup.show(200);
+    var popup = $(POPUP_SELECTOR);
+    if ($(this).hasClass(CONTACT_ACTIVE_CLASS)){
```
Comment (vitali-usik) [inline js/script.js]: Missed space before `{`

File: js/script.js [hunk 3]
```diff
@@ -1,63 +1,85 @@
...
-  fixedNav();
+
+  positionToFixed();
+
+  window.onscroll = positionToFixed;
+
   navList.click(navScroll);
   
   $(CONTACTS_SELECTOR).click(function() {
     var cont = $(CONTACTS_SELECTOR);
```
Comment (vitali-usik) [inline js/script.js]: Is it an array? If it - use contactsArray or similar. If not - use full word 'contact'.

File: js/script.js [hunk 4]
```diff
@@ -1,63 +1,85 @@
...
-    $(this).children().show(100);
+    cont.children().hide();
+    $(this).children().show();
+    }
   });
+
   $(JOIN_BUTTON_ID).click(function() {
     var value = $(EMAIL_INPUT_ID).val();
     console.log(value);
+    $(EMAIL_INPUT_ID).val("");
```
Comment (vitali-usik) [inline js/script.js]: Please use quotes in one style. I advise `''`

File: js/script.js [hunk 5]
```diff
@@ -1,63 +1,85 @@
...
   });
+
   $('input').focus(function(){
     $(this).data('placeholder', $(this).attr('placeholder'))
```
Comment (vitali-usik) [inline js/script.js]: Missed ';'

File: js/script.js [hunk 6]
```diff
@@ -1,63 +1,85 @@
...
+
   $(SLIDER_BUTTON_ID).click(function() {
     var pictureOne =  $(SLIDER_PICTURE_ONE_ID).attr("src");
     var pictureTwo =  $(SLIDER_PICTURE_TWO_ID).attr("src");
     $(SLIDER_PICTURE_ONE_ID).attr("src", pictureTwo);
     $(SLIDER_PICTURE_TWO_ID).attr("src", pictureOne);
   });
-  window.onscroll = fixedNav;
+
+
```
Comment (vitali-usik) [inline js/script.js]: One blank line is extra

File: scss/main/main.scss [hunk 7]
```diff
@@ -171,37 +171,27 @@ header {
...
 
-            &:after {
-              content: "";
-              display: block;
-              width: 66px;
-              height: 24px;
-              position: absolute;
-              top: 6px;
-              border-right: 1px solid #bcbec3;
+              &:before, &:after {
```
Comment (vitali-usik) [inline scss/main/main.scss]: Move '&:after' to separate line

File: scss/main/main.scss [hunk 8]
```diff
@@ -252,16 +238,25 @@ nav {
           padding: 36px 20px;
           font-size: 13px;
           border-right: 1px solid #eceef4;
+          transition: 0.2s;
```
Comment (vitali-usik) [inline scss/main/main.scss:242]: Is this transition works correctly?

**Review by vitali-usik — CHANGES_REQUESTED** [review]

File: scss/main/main.scss (hunk 8 above)
Comment (IgorSinyak) [reply scss/main/main.scss:242]: Yes, it works only for color

**Review by vitali-usik — APPROVED** [review]

**Pull request merged**